### PR TITLE
Simplified regex, improved variable names, made url targetting stricter

### PR DESCRIPTION
**Pull request by AmyMagdalena**

### CODE REVIEW

**Comment by AmyMagdalena** [comment]
This is untested code

File: src/plugins/uwuri_parser.py
Comment (vicky5124) [inline src/plugins/uwuri_parser.py:1]: The plugins folder is for the discord API itself, and should only have events and commands.
The contents of this file can either be added in `src/utils.py` or a different file in `src/.`

**Review by vicky5124 — CHANGES_REQUESTED** [review]

File: src/plugins/fxurl.py [hunk 1]
```diff
@@ -11,44 +10,33 @@
...
-        # The embed in the original message may be delayed. Supress it after the timeout.
-        await asyncio.sleep(10)
+        if url.hostname in {"twitter.com", "www.twitter.com", "mobile.twitter.com"}:
+            fx_url += f"https://fxtwitter.com{url.path}\n"
+        if url.hostname in {"pixiv.net", "www.pixiv.net"}:
+            fx_url += f"https://fxpixiv.net{url.path}\n"
+
+    # If the message had URLs that weren't twitter or pixiv, skip the fixing.
+    if fx_url:
         await msg.edit(flags=MessageFlag.SUPPRESS_EMBEDS)
```
Comment (vicky5124) [inline src/plugins/fxurl.py:38]: This code is important.

```suggestion
        await msg.respond(fx_url, reply=True)
        # The embed in the original message may be delayed. Supress it after the timeout.
        await asyncio.sleep(10)
        await msg.edit(flags=MessageFlag.SUPPRESS_EMBEDS)
```

**Review by vicky5124 — CHANGES_REQUESTED** [review]

File: src/plugins/fxurl.py [hunk 2]
```diff
@@ -11,44 +10,36 @@
...
+        if url.hostname in {"pixiv.net", "www.pixiv.net"}:
+            fx_url += f"https://fxpixiv.net{url.path}\n"
+
+    # If the message had URLs that weren't twitter or pixiv, skip the fixing.
+    if fx_url:
+        await msg.respond(fx_url, reply=True)
         # The embed in the original message may be delayed. Supress it after the timeout.
         await asyncio.sleep(10)
         await msg.edit(flags=MessageFlag.SUPPRESS_EMBEDS)
+        await msg.respond(fx_url, reply=True)
```
Comment (vicky5124) [inline src/plugins/fxurl.py:42]: Duplicate

```suggestion
```

File: src/plugins/fxurl.py [hunk 3]
```diff
@@ -1,6 +1,5 @@
-import asyncio
-import re
 import typing as t
```
Comment (vicky5124) [inline src/plugins/fxurl.py:1]: Missing import.

```suggestion
import asyncio
import typing as t
```

**Review by vicky5124 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**